### PR TITLE
Drum and extended magazines

### PR DESCRIPTION
**Pull request by Humonitarian**
## About The Pull Request

Adds extended .20 mags, .20 drum mags, as well as .30 drum mags.

Replaces 6 of the 18 .30 mags in the serbian ship with drum variants.
Serbian Arms AKs can use the drum variant.
The Wintermute and the Vintorez can use the extended variants.

No gun can use the .20 drum mags at the moment, so it does not spawn either.

## Why It's Good For The Game

Tacticool + cool looking mags + Krinkov sucks

## Changelog
:cl:
add: Two new magazine types: extended .20 magazines for the vintorez and wintermute, as well as .30 drum magazines exclusively for the Krinkov
add: .20 magazines have a new cooler sprite
/:cl:

### CODE REVIEW

**Comment by thedarkcrusader** [comment]
image

**Comment by Humonitarian** [comment]
Missing .30 mags @thedarkcrusader

File: code/modules/projectiles/ammunition/magazines.dm [hunk 1]
```diff
@@ -193,6 +193,71 @@
 	ammo_type = /obj/item/ammo_casing/srifle/scrap
 	spawn_tags = SPAWN_AMMO_COMMON
 
+////////// .20 LONG ///////////
+
+/obj/item/ammo_magazine/srifle/long
+	name = "extended magazine (.20 Rifle)"
```
Comment (Firefox13) [inline code/modules/projectiles/ammunition/magazines.dm:199]:
```suggestion
	name = "extended magazine (.20 Rifle)"
	desc = "Extended .20 caliber magazine, hold 35 rounds. Only fits /"Vintorez/" and /"Wintermute/".
```

File: code/modules/projectiles/ammunition/magazines.dm [hunk 2]
```diff
@@ -193,6 +193,71 @@
...
+	ammo_type = /obj/item/ammo_casing/srifle/rubber
+
+/obj/item/ammo_magazine/srifle/long/scrap
+	ammo_type = /obj/item/ammo_casing/srifle/scrap
+	spawn_tags = SPAWN_AMMO_COMMON
+
+////////// .20 DRUM ///////////
+
+/obj/item/ammo_magazine/srifle/drum
+	name = "drum magazine (.20 Rifle)"
```
Comment (Firefox13) [inline code/modules/projectiles/ammunition/magazines.dm:232]:
```suggestion
	name = "drum magazine (.20 Rifle)"
	desc = "Heavy .20 caliber magazine, holds 60 rounds. Only fits \"Kalash\"."
```

File: code/modules/projectiles/ammunition/magazines.dm (hunk 1 above)
Comment (Humonitarian) [reply code/modules/projectiles/ammunition/magazines.dm:199]: Oh yeah this one fits Kalash too, no?

File: code/modules/projectiles/ammunition/magazines.dm [hunk 3]
```diff
@@ -284,46 +365,63 @@
...
-/obj/item/ammo_magazine/lrifle/pk/mech
-	name = "LMG munitions box (.30 Rifle, Exosuit)"
-	matter = list()
-	spawn_blacklisted = TRUE
-	bad_type = /obj/item/ammo_magazine/lrifle/pk/mech
-
-/obj/item/ammo_magazine/lrifle/pk/empty
-	initial_ammo = 0
+/obj/item/ammo_magazine/lrifle/drum
+	name = "drum magazine (.30 Rifle)"
```
Comment (Firefox13) [inline code/modules/projectiles/ammunition/magazines.dm:379]:
```suggestion
	name = "drum magazine (.30 Rifle)"
	desc = "Heavy .30 caliber magazine, holds 45 rounds. Only fits \"Krinkov\" and \"Kalash\"."
```

**Review by Firefox13 — APPROVED** [review]

**Pull request merged**